pkgs/by-name/sc/sc/package.nix

```
@@ -0,0 +1,48 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  ncurses,
+  yacc,
+  lib,
+}:
+
+stdenv.mkDerivation {
+  pname = "sc";
+  version = "2024-08-15";
+
+  src = fetchFromGitHub {
+    repo = "sc";
+    owner = "n-t-roff";
+    rev = "e029bc0fb5fa29da1fd23b04fa2a97039a96d2ba";
+    hash = "sha256-JQY+ixHL+TpP4YRpgB9GP4jO5+PBMS/v5Ad3Ux0+yuQ=";
+  };
+
+  buildInputs = [ ncurses ];
+
+  nativeBuildInputs = [ yacc ];
```

```suggestion
  nativeBuildInputs = [ bison ];
```
It should probably be bison (GNU yacc) instead of yacc.

---

It didn't work so reverted with new force-push

---

I can build it with bison (you also need to change it above in the inputs yacc -> bison).